lib/bloc/user_profile/user_profile_bloc.dart

```diff
@@ -1,33 +1,74 @@
 import 'dart:async';
 import 'dart:io';
 
+import 'package:breez_translations/breez_translations_locales.dart';
+import 'package:c_breez/bloc/user_profile/default_profile_generator.dart';
 import 'package:c_breez/bloc/user_profile/user_profile_state.dart';
 import 'package:c_breez/models/user_profile.dart';
 import 'package:c_breez/services/breez_server.dart';
 import 'package:c_breez/services/notifications.dart';
+import 'package:fimber/fimber.dart';
 import 'package:hydrated_bloc/hydrated_bloc.dart';
 import 'package:path_provider/path_provider.dart';
 
 const PROFILE_DATA_FOLDER_PATH = "profile";
 
+final _log = FimberLog("UserProfileBloc");
+
 class UserProfileBloc extends Cubit<UserProfileState> with HydratedMixin {
   final BreezServer _breezServer;
   final Notifications _notifications;
 
-  UserProfileBloc(this._breezServer, this._notifications)
-      : super(UserProfileState.initial());
+  UserProfileBloc(
+    this._breezServer,
+    this._notifications,
+  ) : super(UserProfileState.initial()) {
+    stream.listen((profile) {
```

Comment:
Why do we need to listen to the stream? Isn't it enough to look for the current value? e.g `this.state`

Reply:
That would work to fix the profile state at the creation time of the bloc; I did this through the stream to have this fix applied to any state change; if you think it is not necessary I can remove the stream.

Reply:
Stream removed